rosdep/osx-homebrew.yaml

```diff
@@ -154,7 +158,7 @@ graphviz:
 gtest:
   osx:
     homebrew:
-      packages: [gtest]
+      packages: [googletest]
```

Comment:
why is this repeated in base and here? I'm not sure how that works

Reply:
@ivanpauno  I have the same doubt. I see other macOS packages randomly scattering in these two files. Maybe someone should clean all these up.